### ISSUE TITLE
visual content misread by screen readers 

### ISSUE THREAD

**Issue by bpdesigns** [18F/e-QIP-prototype#901]
## **Page title subsection issue for screen readers on ALL pages**
">" is read by screen reader as "greater than" 

so in the below example the screen reader reads "Information about you greater than your contact information"
<img width="318" alt="screen shot 2018-09-27 at 9 57 53 am" src="https://user-images.githubusercontent.com/15329140/46151622-2d15d800-c23d-11e8-8182-4e6997a9297f.png">

_Possible ways to address this could be to make the ">" an image and/or make the screen reader skip reading it._ 

## **"sections complete" feature** 
The "/" is read as slash
the below is read as "three slash one zero sections complete"
<img width="174" alt="screen shot 2018-09-27 at 9 57 58 am" src="https://user-images.githubusercontent.com/15329140/46151850-af060100-c23d-11e8-8c12-f131756ca610.png">

- At the very least it should read the "one zero" as "ten"
- I would also have it read the "slash" as "of" 

_Potentially there could me more consideration about when a screen reader user comes across this content and if it is shown to be confusing in usability tests, I would hide the content._ 

##  **"Scalable Vector Graphic" on Instructions popup window** 
"Scalable Vector Graphic" should be read by screenreader users as "Exit Instructions popup button"
<img width="152" alt="screen shot 2018-09-27 at 10 07 09 am" src="https://user-images.githubusercontent.com/15329140/46152270-ab26ae80-c23e-11e8-86d0-43e00a74dc2f.png">
<img width="1061" alt="screen shot 2018-09-27 at 10 07 17 am" src="https://user-images.githubusercontent.com/15329140/46152323-c85b7d00-c23e-11e8-84ad-39a4f98ab1f1.png">

##  **Text elements as visual icons**
Should the "(", ")" , "-" , "#" be read as text in the below example? 
<img width="403" alt="screen shot 2018-09-27 at 9 58 04 am" src="https://user-images.githubusercontent.com/15329140/46152397-ef19b380-c23e-11e8-9dbf-81acfd67c71b.png">
_I recommend usability testing this to see if this is helpful, confusing or unnecessary for screen reader users._ 

## **Warning icons** 
There are either read as "scalable vector graphics", "5" or ignored by screenreaders
<img width="239" alt="screen shot 2018-09-27 at 10 27 12 am" src="https://user-images.githubusercontent.com/15329140/46152892-fa211380-c23f-11e8-88ee-4e524fcab680.png">

_I recommend having them alway read as "Error" by screen readers_
